### ISSUE TITLE
yt 4-0 running list of issues

### ISSUE THREAD

**Issue by dnarayanan** [dnarayanan/powderday#16]
first, this PR is a useful one:

https://github.com/yt-project/yt/pull/1889/files#diff-3366081d2fd998c5503d1a478d8b2d90

with  code snippets that are valuable as:

```
In [16]: octree.keys()
Out[16]: dict_keys(['x', 'y', 'z'])
```

to get positions, and

```
density = octree[('PartType0', 'density')]```

to get the *smoothed* density

**Comment by dnarayanan** [comment]
installation notes for yt4-0:

https://nbviewer.jupyter.org/url/trident-project.org/notebooks/trident_demesh_install.ipynb

**Comment by dnarayanan** [comment]
when we call ds.octree

we should enable the flag ```force_build = True``` per:

https://github.com/yt-project/yt/commit/10116538aa2c27f7bd7e9655ae985ccf265f7e0e

in order to force the octree to rebuild every time. otherwise if we change n_ref it won’t actually rebuild the octree. will be slower but so be it.

**Comment by dnarayanan** [comment]
the octree built by `yt` appears to be breadth first and not depth first.

**Comment by dnarayanan** [comment]
closed by 59315f311535b5f2309c705f5a71519148aa4f29
